components/builder-scheduler/src/config.rs

```diff
@@ -30,6 +30,8 @@ pub struct Config {
     pub worker_threads: usize,
     /// Path to packages on-disk (for migration)
     pub migration_path: String,
+    /// Path to scheduler event logs
+    pub log_path: String,
```

Comment:
I would use `PathBuf` instead of `String` to represent file paths. I've been cleaning this up as I go throughout the codebase